scraper/factory.go

```diff
@@ -35,9 +35,18 @@ type Factory interface {
 	// Implementers can assume `next` is never nil.
 	CreateMetrics(ctx context.Context, set Settings, cfg component.Config) (Metrics, error)
 
+	// CreateLogs creates a Logs scraper based on this config.
+	// If the scraper type does not support logs,
+	// this function returns the error [pipeline.ErrSignalNotSupported].
+	// Implementers can assume `next` is never nil.
```

Comment:
sorry, what is `next` here?

Reply:
```suggestion
```

Reply:
@atoulme Good catch! I think this line is copied from somewhere. Removed it.